WordPress/Classes/ViewRelated/Blog/My Site/NoSitesView.swift

```diff
@@ -0,0 +1,188 @@
+import SwiftUI
+
+protocol NoSitesViewDelegate: AnyObject {
+    func didTapAccountAndSettingsButton()
+}
+
+struct AddNewSiteConfiguration {
+    let canCreateWPComSite: Bool
+    let canAddSelfHostedSite: Bool
+    let launchSiteCreation: () -> Void
+    let launchLoginForSelfHostedSite: () -> Void
+}
+
+struct NoSitesView: View {
+
+    @SwiftUI.Environment(\.colorScheme) var colorScheme
+
+    @State private var isShowingDialog = false
+
+    let viewModel: NoSitesViewModel
+    let addNewSiteConfiguration: AddNewSiteConfiguration
+
+    weak var delegate: NoSitesViewDelegate?
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            Color(uiColor: .listBackground)
+                .edgesIgnoringSafeArea(.all)
```

Comment:
Does this adapt to the theme change? There used to be a problem with `Color:uiColor` init that it lost dynamic nature but rather simply initiated a static color.

Reply:
It does!

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/30f18f58-ac56-48a6-b37a-5cfaebf7843e" width=200> | <img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/206b934f-27c8-4363-88ac-0e84c94c1772" width=200>
-- | --